src/kaleidoscope/plugin/LEDControl/LED-Off.h

```diff
@@ -27,7 +27,8 @@ namespace plugin {
 //
 class LEDOff : public LEDMode {
  public:
-  LEDOff(void) {}
+  LEDOff(void) { ledModeName = "LEDOff"; }
```

Comment:
In all led modes that didn't have a constructor yet, I added constructors as `<class_name>()` rather than `<class_name>(void)`. I couldn't find anything about it in the code style docs and I believe I've seen zero-argument constructors w/o `void` elsewhere.
Should I add the `void` everywhere else? Should I remove it here?

Reply:
I *think* the idea is that the "void" just improves clarity. But it does appear that we use 'without void' more often than 'with void'.   I think removing it here is ok.